### ISSUE TITLE
Preserving spaces around variables

### ISSUE THREAD

**Issue by animify** [ThisIsManta/stylus-supremacy#21]
Would you consider adding an option for preserving spaces around variables or comments between variables?

![image](https://user-images.githubusercontent.com/10724744/35196070-ea91d95a-fed5-11e7-8510-971a096200b6.png)

Preserving whitespace around something like the above would be cool!

**Comment by ThisIsManta** [comment]
You mean a new line before a comment, don't you?

**Comment by animify** [comment]
Yeah before comments, but even just between variables, so that's easier to distinguish groups of variables. Like above, the new lines act as a divider between family/headings/size/weights etc.

**Comment by ThisIsManta** [comment]
I put this on-hold because I still can't find a good solution to parse single-line comments.

**Comment by animify** [comment]
Fair enough, how about an option for now that will preserve new lines around variables?

**Comment by kbtzr** [comment]
I've been using this formatter for a while now and the only thing I see left to make it perfect is to not remove blank lines around grouped variable declarations such as hashes. 

@ThisIsManta Thanks for you amazing work! :pray:
